library/roc_consumer/src/roc_api.e

```diff
@@ -0,0 +1,76 @@
+note
+	description: "ROC CMS HAL API"
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=ROC CMS REST API", "src=https://docs.google.com/document/d/1DYpX46Urp1XC1g5KTevl_qroCumNEFCgYSfTw5to2_4/edit", "protocol=uri"
+class
+	ROC_API
+
+create
+	make
+
+feature {NONE} -- Initialization
+
+	make (a_base_uri: READABLE_STRING_8; a_username: READABLE_STRING_32; a_password: READABLE_STRING_32)
+			-- Create an object roc with a base uri `a_base_uri' user `a_username' and  password `a_password'.
+		do
+			create roc_admin.make (a_base_uri, a_username, a_password)
+		end
+
+	roc_admin: ROC_ADMIN_API
+			-- Admin API.
+
+feature -- Admin: Roles
+
+	roles: STRING
```

Comment:
For me, the redirection is useless
I prefer to call directoy

  roc_api.roc_admin.roles
rather than
   roc_api.roles

and even ... rename roc_admin by admin_api
I think this is how it is done for CMS_API.users_api